documentation-processor/src/main/java/com/koval/resolver/processor/documentation/convert/FileConverter.java

```diff
@@ -1,5 +1,10 @@
 package com.koval.resolver.processor.documentation.convert;
 
+import com.koval.resolver.processor.documentation.bean.MediaType;
+
 public interface FileConverter {
   void convert(String inputFilePath, String outputFilePath);
+
+  Boolean isConvert(MediaType docType);
```

Comment:
What is this method for? I did not see the use of its implementations in classes.